src/README.md

```diff
@@ -49,3 +49,24 @@ const App = () => (
 It'll grab **Open Sans** from Google's CDN [through a CSS import](https://github.com/zopaUK/react-components/blob/master/src/components/styles/Fonts.tsx#L3-L5).
 
 You're free to use this technique or add the dependency manually to your base HTML `<head />`
+
+### Spacing
+
+```ts static
+import { Margin } from '@zopauk/react-components';
+
+// root component
+const App = () => (
+  <>
+    <Margin />
+    <Padding />
+    // rest of your top-level components
+  </>
+);
+```
```

Comment:
Could we remove this code example? I think it would tempt people to do it that way rather than rely on `<GlobalStyles />` 💆🏻‍♂️

I also suggest we add a **table** here with all the available `maring`and `padding` classes as a reference for people. In case that's too much work we can add a [link to tailwind](https://tailwindcss.com/docs/margin/#app)?

Reply:
that was something I was planning to do this is still very much WIP.
Will update soon!